### ISSUE TITLE
pretrained weights?

### ISSUE THREAD

**Issue by lly456** [Event-AHU/OpenPAR#37]
where can fond the pretrained weights? such as Epoch60.pth?

**Comment by 1125178969** [comment]
denote the LLM-PAR or which works?

**Comment by cbiras** [comment]
Yes, how can we get the pretrained weights for LLM-PAR? @NOP1224

**Comment by NOP1224** [comment]
ok we will upload the pretrained weights for LLM-PAR in recent @cbiras

**Comment by cbiras** [comment]
I see that you provided PETA and RAP petrained weights. Is there a way you could upload them to google drive? Baidu download seems to not be working @NOP1224

**Comment by wangxiao5791509** [comment]
@cbiras  Hi, we update the DropBox URLs for the two checkpoints, you can find them in the readme.md 

<img width="500" alt="Image" src="https://github.com/user-attachments/assets/7157a8a8-7c3a-49c0-9ad5-807cf63ca92e" />
